app/javascript/components/exercise/TheProblems.js

```diff
@@ -0,0 +1,85 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import ProblemContents from './ProblemContents'
+
+class TheProblems extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      visibleProblemIndex: 0
+    }
+  }
+  renderTabs = (visibleProblemIndex) => {
+    return (
+      <div className='tabs'>
+        {
+          this.props.problemList.map((p, i) => (
+            this.renderTab(i, visibleProblemIndex)
+          ))
+        }
+      </div>
+    )
+  }
+
+  renderTab = (index, visibleProblemIndex) => {
+    if ( index === visibleProblemIndex) {
+      return (
+        <li
+          className={index === visibleProblemIndex && 'is-active'}
+          key={index}
+          onClick={() => {this.changeActiveTab(index)}}
+        >
+          <a>{`problem${index+1}`}</a>
+        </li>
+      )
+    }
+    return (
+      <li
+        key={index}
+        onClick={() => {this.changeActiveTab(index)}}
```

Comment:
**[eslint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Replace `this.changeActiveTab(index)}}` with `⏎··········this.changeActiveTab(index)` (prettier/prettier)